.gitignore

```diff
@@ -167,3 +167,5 @@ modules.xml
 
 .env
 .env.template
+*coverage*
+.gitignore
```

Comment:
Why?

Reply:
Sorry about that, the PR is still a draft for now, that will be cleaned up tomorrow